core/src/main/res/values/arrays.xml

```diff
@@ -1,44 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <string-array name="enc_method_entry" translatable="false">
-        <item>RC4-MD5</item>
-        <item>AES-128-CFB</item>
-        <item>AES-192-CFB</item>
-        <item>AES-256-CFB</item>
-        <item>AES-128-CTR</item>
-        <item>AES-192-CTR</item>
-        <item>AES-256-CTR</item>
-        <item>BF-CFB</item>
-        <item>CAMELLIA-128-CFB</item>
-        <item>CAMELLIA-192-CFB</item>
-        <item>CAMELLIA-256-CFB</item>
-        <item>SALSA20</item>
-        <item>CHACHA20</item>
-        <item>CHACHA20-IETF</item>
+        <item>PLAIN</item>
         <item>AES-128-GCM</item>
-        <item>AES-192-GCM</item>
         <item>AES-256-GCM</item>
         <item>CHACHA20-IETF-POLY1305</item>
         <item>XCHACHA20-IETF-POLY1305</item>
     </string-array>
 
     <string-array name="enc_method_value" translatable="false">
-        <item>rc4-md5</item>
-        <item>aes-128-cfb</item>
-        <item>aes-192-cfb</item>
-        <item>aes-256-cfb</item>
-        <item>aes-128-ctr</item>
-        <item>aes-192-ctr</item>
-        <item>aes-256-ctr</item>
-        <item>bf-cfb</item>
-        <item>camellia-128-cfb</item>
-        <item>camellia-192-cfb</item>
-        <item>camellia-256-cfb</item>
-        <item>salsa20</item>
-        <item>chacha20</item>
-        <item>chacha20-ietf</item>
+        <item>plain</item>
```

Comment:
We might want to keep the old ciphers for backwards compatibility...

Reply:
Integrating openssl is much difficult than I thought. Is it acceptable to use prebuilt binaries?

Reply:
binaries for openssl? We can definitely use it as a WIP and worry about it later.

Reply:
OpenSSL also works now by enabling `opnessl-vendored` feature.

I didn't expect we can integrate openssl in that way... It's really cool.

Reply:
Isn't modern dependency management amazing? (except when they don't work)